get-started/mobile-traffic/configure-android.md

```diff
@@ -9,55 +9,54 @@ position: 10
 previous_url: /knowledge-base/configure-android
 ---
 
-#### Environment
+# How to Capture and Inspect Android Traffic
+
+You can use Fiddler Everywhere to capture and inspect traffic that comes from Android devices and emulators. This article describes the required steps.
+
+>caption Environment
 
 |   |   |
 |---|---|
 | Product  | Fiddler Everywhere  |
 | Product Version | 1.0.2 and above  |
 
-#### Description
-
-Fiddler Everywhere can be used to capture and inspect traffic that comes from Android devices and emulators.
-
 ## Prerequisites
 
 - [Fiddler Everywhere client](https://www.telerik.com/download/fiddler-everywhere) installed on your machine.
-- The computer (that holds Fiddler Everywhere) and the Android device should be **discoverable** on the same network.
+- The machine running Fiddler Everywhere and the Android device must be **discoverable** on the same network, for example, 192.168.0.0
```

Comment:
Can we link a good definition of _discoverable_? If there is no appropriate link, we need to define the term.